controllers/tf_controller_plan.go

```diff
@@ -59,6 +59,12 @@ func (r *TerraformReconciler) plan(ctx context.Context, terraform infrav1.Terraf
 		planRequest.Destroy = true
 	}
 
+	// check if Lock Timeout is set
+	if terraform.Spec.LockTimeout != "0s" {
+		log.Info("lock timeout is set")
+		planRequest.LockTimeout = terraform.Spec.LockTimeout
```

Comment:
Don't forget to update this :slightly_smiling_face:

Reply:
trying to wrap my head around how testing works :)